R/graded.R

```diff
@@ -90,3 +234,25 @@ legacy_graded <- function(...) {
     graded(...)
   )
 }
+
+get_from_env <- function(x, env) {
+  if (exists(x, envir = env)) {
+    get(x, envir = env)
+  } else {
+    rlang::missing_arg()
+  }
```

Comment:
This should be the same:

```suggestion
  get0(x, envir = env, ifnotfound = rlang::missing_arg())
```